src/markdown-to-inner-partitions.test.js

```diff
@@ -0,0 +1,40 @@
+import { generateInnerPartitions } from './markdown-to-inner-partitions';
+
+let testCases = [
+  // {
+  //   name: 'test for bold',
+  //   paramMarkdown: '*This is a test for bold partitions*',
+  //   returnPartitions: [{type: 'bold', value: 'This is a test for bold partitions'}]
+  // },
+  // {
+  //   name: 'test for italics',
+  //   paramMarkdown: '_This is a test for italics partitions_',
+  //   returnPartitions: [{type: 'italics', value: 'This is a test for italics partitions'}]
+  // },
```

Comment:
Are these two zombie code or should they be uncommented?

Reply:
Oh I commented them for debugging they should be uncommented.